cmd/validator.go

```diff
@@ -145,26 +147,26 @@ var validatorUpdateCmd = &cobra.Command{
 			if err != nil {
 				log.Fatal(err)
 			}
-			validator.MaxStake = stake
+			updateValidator.MaxStake = &stake
 		}
 
 		if flags.Changed("num_delegates") {
 			var nd int
 			if nd, err = flags.GetInt("num_delegates"); err != nil {
 				log.Fatal(err)
 			}
-			validator.NumDelegates = nd
+			updateValidator.NumDelegates = &nd
 		}
 
 		if flags.Changed("service_charge") {
 			var sc float64
 			if sc, err = flags.GetFloat64("service_charge"); err != nil {
 				log.Fatal(err)
 			}
-			validator.ServiceCharge = sc
+			updateValidator.ServiceCharge = &sc
```

Comment:
Will the update works if I only update part of the stake pool settings, such as `MinStake`. I ask because I find the changes in gosdk will only update the stake pool settings when all fields are set.

Reply:
From gosdk, the intended behaviour is to set the sp when atleast one of the field is set. So if only `MinStake` is changing it will be set.